### PR TITLE
Parse Path that Includes URL

### PR DESCRIPTION
**Pull request by amayer-hc**
This allows assets to be hosted on a CDN.

### CODE REVIEW

File: src/Mix.php [hunk 1]
```diff
@@ -27,6 +27,12 @@ class Mix
      */
     public function __invoke($path, $manifestDirectory = ''): string
     {
+        $urlDomain = '';
+        if (preg_match('@^([a-z]+://[^/]+)(/.+)$@', $path, $matches)) {
```
Comment (ishanvyas22) [inline src/Mix.php:31]: Can you please add a comment saying what this regex is doing? Also, it would be great if you also add a https://regex101.com/ URL.

**Review by ishanvyas22 — CHANGES_REQUESTED** [review]
Overall looks good. Just a minor thing to take care of.

File: src/Mix.php (hunk 1 above)
Comment (amayer-hc) [reply src/Mix.php:33]: Comment added.

**Review by ishanvyas22 — APPROVED** [review]

**Comment by ishanvyas22** [comment]
Thanks!

**Pull request merged**